js/client-teambuilder.js

```diff
@@ -1289,12 +1289,14 @@
 					success: function (data) {
 						if (/^https?:\/\/pokepast\.es\/.*\/json\s*$/.test(url)) {
 							var teamData = JSON.parse(data);
-							Storage.activeSetList = self.curSetList = Storage.importTeam(teamData.paste);
+							var format = window.BattleFormats && window.BattleFormats[toID(teamData.notes.slice(8))];
```

Comment:
It's probably worth checking if the first 8 characters of `teamData.notes` is in fact `"Format: "`.

Also, can we slice it only to the next linebreak? I think both of these are probably useful future-proofing.